pyqgiswps/executors/io/filesio.py

```diff
@@ -128,18 +129,35 @@ def parse_output_definition(outdef: QgsProcessingOutputDefinition, kwargs,
     elif isinstance(outdef, QgsProcessingOutputFile):
         # Try to get a corresponding inputFileDefinition
         # See https://qgis.org/pyqgis/master/core/QgsProcessingParameterFileDestination.html
-        mime = None
+        supported_formats = []
         if alg:
             inputdef = alg.parameterDefinition(outdef.name())
             if isinstance(inputdef, QgsProcessingParameterFileDestination):
-                mime = mimetypes.types_map.get("." + inputdef.defaultFileExtension())
+                default_extension = inputdef.defaultFileExtension()
+                default_extension_idx = -1
+                for filter_ in inputdef.fileFilter().split(";;"):
+                    extension_regex = QRegularExpression("[*.]([a-z]+)")
```

Comment:
Why using a `QRegularExpression` and not and  python re ?

Reply:
no reason. Changed to `re`.